test/terraform/budget/variables.tf

```diff
@@ -32,3 +32,7 @@ variable "budget_alert_rule_recipients" {
   type        = string
   default     = "example@example.com"
 }
+variable "home_compartment_id" {
```

Comment:
We need to get this from somewhere.  We might need to create a  home compartment to put the tested module in.